src/main/java/cloud/eppo/EppoClient.java

```diff
@@ -26,12 +25,14 @@ public class EppoClient {
   private static final String DEFAULT_HOST = "https://fscdn.eppo.cloud";
   private static final boolean DEFAULT_IS_GRACEFUL_MODE = true;
 
+  private final ConfigurationStore configurationStore;
   private final ConfigurationRequestor requestor;
   private final AssignmentLogger assignmentLogger;
-  private final boolean isGracefulMode;
+  private final BanditLogger banditLogger;
   private final String sdkName;
   private final String sdkVersion;
   private final boolean isConfigObfuscated;
+  private boolean isGracefulMode;
```

Comment:
made non-final so that graceful mode can be toggled at runtime. Useful for tests, and possibly for developers too.